lib/thesis/stores/store.ex

```diff
@@ -20,14 +20,14 @@ defmodule Thesis.Store do
 
   @doc """
   Updates the given page (identified by its slug) with the given map of
-  string keys and Thesis.PageContent structs. Returns `:ok`.
+  string keys and Thesis.PageContent structs. Returns tuple `{:ok, page}`.
```

Comment:
These are breaking changes. But I think we should do it now, for sure. It'll be harder to change later.